src/main/java/Ex14.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;
+
+public class Ex14 {
+    public int demSoChuSo(int n) {
+        int m = 0;
+        while (n > 0) {
+            n = n / 10;
+            m++;
+        }
+        return m;
+    }
+
+    public boolean checkArmstrong(int n) {
+        int sum = 0;
+        int digits = demSoChuSo(n);
+        int p = n;
+
+        for (int i = 1; i <= digits; i++) {
+            int z = p % 10;
+            sum += Math.pow(z, digits);
```

Comment:
Biến z thật ra không cần thiết lắm, vấn đề là khai báo tên thế này không meaningful

```suggestion
            sum += Math.pow(p % 10, digits);
```

Reply:
Done